### DISCUSSION TITLE
Tool to analyze target image trackability

### DISCUSSION THREAD

**Discussion by hiukim** [AR-js-org/AR.js#328] [Show and tell]
Sharing a tool I made to analyze the quality of images in terms of detection and tracking.

https://pictarize.com/image-analyzer/

I also wrote an article series to explain how tracking works behind the scene, and how to choose target image wisely. 
https://blog.pictarize.com/how-to-choose-a-good-target-image-for-tracking-in-ar-part-1/

![image-analyzer](https://user-images.githubusercontent.com/459126/130394878-24629bd5-9942-44a5-97b2-d7a2e74343d6.png)

**Comment by kalwalt** [comment]
Very nice and useful tool @hiukim !! Thanks for sharing it!
